libbeat/common/cfgwarn/removed.go

```diff
@@ -0,0 +1,48 @@
+package cfgwarn
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/joeshaw/multierror"
+)
+
+func CheckRemoved5xSettings(cfg *common.Config, settings ...string) error {
```

Comment:
Yeah, it's not the only time we remove settings. But at some point in time I want to remove the messages (e.g. on 7.x release remove the 5.x checks). Being able to remove messages/warnings  is the motivation for naming this 5x